yt_dlp/extractor/rtrfm.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class RTRFMIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?rtrfm\.com\.au/(?:shows|show-episode)/(?P<id>[^/?\#&]+)'
+    _PLAY_SHOW = r"\.playShow\('(?P<show>[^']+)', *'(?P<date>[0-9]{4}-[0-9]{2}-[0-9]{2})', *'(?P<title>[^']+)'\)"
+    _PLAY_SHOW_FROM = r"\.playShowFrom\('(?P<show>[^']+)', *'(?P<date>[0-9]{4}-[0-9]{2}-[0-9]{2})', *'?P<title>([^']+)', \d+\)"
+    _RESTREAMS_URL = 'https://restreams.rtrfm.com.au/rzz'
+    _TESTS = [
+        {
+            'url': 'https://rtrfm.com.au/shows/breakfast/',
+            # the downloaded file changes daily so the md5 is not checkable
+            'info_dict': {
+                'id': r're:^breakfast-\d{4}-\d{2}-\d{2}$',
+                'ext': 'mp3',
+                'series': 'Breakfast with Taylah',
+                'title': r're:^Breakfast with Taylah \d{4}-\d{2}-\d{2}$',
+                'description': 'md5:0979c3ab1febfbec3f1ccb743633c611',
+            },
+            'skip': 'tests with re in the id expect the re in the filename '
+                    'instead of the id, but the id depends on the date.',
+        },
```

Comment:
Dont use `re:` in id. Just add a test that is passes atm, and then just add something like `'skip': 'ID keeps changing'`

Reply:
Done.